digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/linearasset/GraphPartitioner.scala

```diff
@@ -8,8 +8,8 @@ import org.geotools.graph.structure.basic.BasicEdge
 import scala.collection.JavaConversions._
 
 trait GraphPartitioner {
-  protected def clusterLinks[T <: PolyLine](links: Seq[T]): Seq[Graph] = {
-    val generator = new BasicLineGraphGenerator(0.5)
+  protected def clusterLinks[T <: PolyLine](links: Seq[T], tolerance: Double = 0.5): Seq[Graph] = {
```

Comment:
maybe add TODO here so we remember to use tolerance from easily accessible place like viite\package.scala  MaxSuravageToleranceToGeometry, and later remove default value from this trait

Reply:
GraphPartitioner is a common trait for both OTH and Viite use - so default value of 0.5 is set as default here.

Reply:
yea, thought it would be nice to have oth and viite side their own single file that defines different tolerances and use those in case we ever have to change it.